### PR TITLE
Added an icon to indicate if folders are empty

### PR DESCRIPTION
**Pull request by winiston2212**
Adds and uses new icon for non-empty folders as shown in issue #1574.

image

### CODE REVIEW

**Comment by XPoppyX** [comment]
the problem with the first design I proposed, it doesn't look good in light mode
Anotación 2020-07-30 184204

What about the second proposal?

Anotación 2020-07-30 185118

**Comment by winiston2212** [comment]
@XPoppyX Looks great. I'll update it now.

**Comment by XPoppyX** [comment]
folder shortcuts icons do not change
Anotación 2020-07-30 200025

**Comment by winiston2212** [comment]
@XPoppyX How are you getting shortcuts? They don't show for me.

**Comment by XPoppyX** [comment]
are you sure your branch has the latest changes?

File: Files/Filesystem/ListedItem.cs [hunk 1]
```diff
@@ -13,8 +14,29 @@ public class ListedItem : ObservableObject
...
+        public bool ContainsFilesOrFolders { get; set; }
         private bool _LoadFileIcon;
 
+        public Uri FolderIconSource
+        {
+            get
+            {
+                if (ContainsFilesOrFolders)
+                    return new Uri("ms-appx:///Assets/FolderIcon2.svg");
+                else return new Uri("ms-appx:///Assets/FolderIcon.svg");
```
Comment (yaichenbaum) [inline Files/Filesystem/ListedItem.cs:26]: Can you please format the code properly?

File: Files/Filesystem/ListedItem.cs [hunk 2]
```diff
@@ -13,8 +14,29 @@ public class ListedItem : ObservableObject
...
+
+        public Uri FolderIconSourceLarge
+        {
+            get
+            {
+                if (ContainsFilesOrFolders)
+                    return new Uri("ms-appx:///Assets/FolderIcon2Large.svg");
+                else return new Uri("ms-appx:///Assets/FolderIconLarge.svg");
+            }
+        }
```
Comment (yaichenbaum) [inline Files/Filesystem/ListedItem.cs:38]: Can you please format the code properly?

**Pull request merged**